src/pages/sign-up/index.js

```diff
@@ -4,17 +4,36 @@ import Button from '../../components/button'
 import { useState } from 'react';
 import '../../App.css'
 
-function App() {
+
+
+
+  function App(props) {
   
   const [input, setInput] = useState('')
   const [inputTwo, setInputTwo] = useState('')
+  
+  const onClick = () => {
+    props.history.push('/home')
+  }
+
+  const submitter = () => {
+   return (
+   <div>
+   {input.length===0 || inputTwo.length===0 ? null : onClick() }
+  </div>
+   )
+}
+
+
 
   return (
-    <div style={{ textAlign: 'center', display: 'flex', justifyContent: 'center', height: '100vh', flexDirection: 'column' }}>
-      <p>Sign Up</p>
-      <Input  textSetter={(event) => setInput(event.target.value)} placeholder={'input 1'}   />
-      <Input textSetter={(event) => setInputTwo(event.target.value)} placeholder={'input 2'}   />
-      <Button  buttonName={'Button Test'} submit={() => console.log(input, inputTwo)}/>
+    <div className='Background' >
+      <h1>Please sign up to get started!</h1>
+      <div style={{alignSelf: 'center', textAlign: 'center', display: 'flex', justifyContent: 'center', height: '50vh', width: '50vh', flexDirection: 'column', backgroundColor: '#F59606', borderRadius: '20px', gap: '20px' }}>
```

Comment:
Can we move all the inline styles into a new css file just for the signup page

Reply:
also done, there is now a "sign-up.css' file in the sign-up folder which is styling the orange <div>.